product_documentation_set/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2019 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Product Documentation Sets",
+    "summary": """
+        Manage Product Documentation""",
+    "version": "12.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "Open Source Integrators,Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/product-attribute",
+    "depends": ["product"],
```

Comment:
```suggestion
    "depends": ["product", "sale"],
```

Reply:
I'm working on a second module that will use Documentation Sets for Sale Orders. Still WIP.

Reply:
Thanks for your clarification. Should we improve module usage section ? This will help to functional review.

Reply:
OK